src/pages/workspace/companyCards/WorkspaceCompanyCardsListRow.tsx

```diff
@@ -55,6 +58,15 @@ function WorkspaceCompanyCardsListRow({cardholder, name, cardNumber}: WorkspaceC
                     {cardNumber}
                 </Text>
             </View>
+            <View style={[styles.justifyContentCenter, styles.alignItemsCenter, styles.ml2]}>
```

Comment:
You forgot to consider the hover case here which caused https://github.com/Expensify/App/issues/72565